tests/test_actions.py

```diff
@@ -442,22 +448,22 @@ def test_implements_protocol(self) -> None:
         assert isinstance(e, Describable)
         assert isinstance(e, Chainable)
 
-    def test_perform_enter2fatoken(self, Tester) -> None:
+    def test_perform_enter2fatoken(self, Tester: Actor) -> None:
         target, element = get_mocked_target_and_element()
         mfa_token = "12345"  # The kind of thing an idiot would have on his luggage!
-        mocked_2fa = Tester.ability_to(AuthenticateWith2FA)
+        mocked_2fa = cast(mock.Mock, Tester.ability_to(AuthenticateWith2FA))
```

Comment:
`cast`, you can do that in Python??

Reply:
Yeah.  It always feels like a cheat, but this is one of those cases it makes sense to use it.